### PR TITLE
add License field + Copyright section to all CPSs

### PR DESCRIPTION
**Pull request by rphair**
Implements change agreed in #762 for all merged CPSs.  These all add CC licensing (vs. Apache, so far) since, as mainly will be true for CPSs, they're all narrative & no code.

This is also a check to see if there will be any downstream problems, mainly due to the new mandatory field in the YAML preamble, on (cc @fill-the-fill @katomm):
- `cips.cardano.org` ([cf-cip-frontend](https://github.com/cardano-foundation/cf-cip-frontend))
- the CIP / CPS build script on the Dev Portal

TODO wait before merging to confirm no downstream problems before:
- [ ] merging this PR
- [ ] updating PRs for other CPSs in progress

### CODE REVIEW

**Comment by Ryun1** [comment]
Pleased to see this
will wait on confirmation from downstream tooling developers that they are aware, before I approve

**Comment by rphair** [comment]
Note I've just added the license stuff to CIPs 11 & 13 (merged earlier today) in commit 8c244a8 but they don't appear in the PR thread or diffs yet (i.e. the sometime "git PRs out of sync" issue which sometimes last a few hours, very rarely).  Rest assured they are there & they can be seen in the PR branch. 😇

**Comment by rphair** [comment]
Question to pre-empt any difficulties has been more officially presented to the site building team in https://github.com/cardano-foundation/developer-portal/issues/1221 ... if not hearing for another week then I'd say we can go ahead and merge this since my educated guess is that the extra field & section will be ignored.  And if they _were_ to cause site building errors then we would have an additional imperative to get the build scripts updated quickly. 😇

**Review by Ryun1 — APPROVED** [review]
Nice improvement ✅

**Comment by rphair** [comment]
thanks & without any objection from the CF Dev Portal representatives (@fill-the-fill @katomm), as well as a preliminary assessment that we didn't have a Dev Portal build failure the last time we had conditions like this (https://github.com/cardano-foundation/developer-portal/issues/1221#issuecomment-1973574971), I think we have done due diligence before merging this change.

Any opinions otherwise, please blow the whistle now... otherwise I'll merge this as soon as we get a another confirmatory review from @Crypto2099 (cc @KtorZ @SebastienGllmt).

File: CPS-0005/README.md [hunk 1]
```diff
@@ -167,3 +168,7 @@ Many of the above problems could be mitigated with a good metadata solution, and
 
 For example, simply knowing the script itself (i.e. the pre-image of the hash used in the script address) helps with problem 1, because then you can know that it’s a Plutus script.
 But it still doesn’t tell you what the form of the datum should be (problem 6), but this could be conveyed with additional metadata.
+
+## Copyright
+
+This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```
Comment (Crypto2099) [inline CPS-0005/README.md:174]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

???

File: CPS-0007/README.md [hunk 2]
```diff
@@ -97,3 +98,7 @@ Below are a set of philosophical questions applicable across a broad range of so
 4. What kinds of community standards are needed outside of the scope of a specific solution at the ledger layer?
 5. What kinds of incentives (or disinsentives) should we use to align behavior with our goals? Should we try to compensate actors for the time and effort they put in, or does that create corrupt and misaligned incentives of their own?
 6. Is the ability to delegate overall more helpful or harmful? Does the risk of "popularity" based delegation (as opposed to trust / expertise based delegation) dilute the votes of others more than it increases inclusivity?
+
+## Copyright
+
+This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```
Comment (Crypto2099) [inline CPS-0007/README.md:104]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

File: CPS-0010/README.md [hunk 3]
```diff
@@ -354,3 +355,7 @@ What options do we have for integration with Cardano sidechains, rollups, hardfo
 
 ### How can we effectively police API scope?
 How can we prevent duplication of functionality and make sure different APIs do not overlap?
+
+## Copyright
+
+This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```
Comment (Crypto2099) [inline CPS-0010/README.md:361]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

File: CPS-0011/README.md [hunk 4]
```diff
@@ -62,3 +63,7 @@ Non-goals:
 - How can we programmatically or manually test the constructed schema file?
 - How to encode long integer arithmetic? Some JSON encoding implementations simply refuse to handle long integers, e.g. [CSL](https://github.com/Emurgo/cardano-serialization-lib/blob/4a35ef11fd5c4931626c03025fe6f67743a6bdf9/rust/src/plutus.rs#L1370).
 - [RFC8949](https://www.rfc-editor.org/rfc/rfc8949.html#name-converting-data-between-cbo) and [RFC8610](https://datatracker.ietf.org/doc/html/rfc8610#appendix-E) contain recommendations for developers who want to maintain interoperability with JSON. Can we apply these in our context?
+
+## Copyright
+
+This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```
Comment (Crypto2099) [inline CPS-0011/README.md:69]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

File: CPS-0013/README.md [hunk 5]
```diff
@@ -100,3 +100,7 @@ Since map operations currently have much worse complexity than a good map data s
 ## References
 
 - https://x.com/Quantumplation/status/1733298551571038338?s=20
+
+## Copyright
+
+This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```
Comment (Crypto2099) [inline CPS-0013/README.md:106]:
```suggestion
This CPS is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```

**Review by Crypto2099 — APPROVED** [review]
Probably a quick oversight from copying from the CIP template, but update CIP == CPS in the Copyright block but otherwise everything looks good to me!

**Comment by rphair** [comment]
thanks @Crypto2099 for catching the above... I'm grateful not to have left people wondering about that & then have had to go back & fix all of them later 🥴

**Pull request merged**